rpm run login - cannot find module (*.ts)

**Description**
It's giving an error when I try to run "npm run login":
```
Error: Cannot find module '../utils/store-instance'
Require stack:
C:\git\hubs\src\utils\phoenix-utils.js
C:\git\hubs\scripts\login.js
    at Object.<anonymous> (C:\git\hubs\src\utils\/phoenix-utils.js:5:1)
    at Generator.next (<anonymous>) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\git\\hubs\\src\\utils\\phoenix-utils.js',
    'C:\\git\\hubs\\scripts\\login.js'
  ]
}
```

I believe the problem is occurring on .ts files. They were not compiled on login command

**To Reproduce**
Steps to reproduce the behavior:
1. Clone repo
2. run `npm ci`
3. run `npm run login`

**Expected behavior**
Run the command without errors

**Hardware** <!-- Please complete the following information -->
 - Device: Desktop
 - OS: Windows
 - Browser: N/A

Hello Users

I too have the same issue and I’m new to Hubs cloud. I have a working AWS Hubs Cloud instance and am trying to setup the custom client and seeing MODULE_NOT_FOUND error during 'npm run login'. PFB error.

Thank you!

$ npm run login

hubs@0.0.1 login
node -r @babel/register -r esm -r ./scripts/shim scripts/login.js
..
..
..

usr/bin/hubs-master/hubs/src/utils/phoenix-utils.js:1
Error: Cannot find module '../utils/store-instance'
Require stack:
usr/bin/hubs-master/hubs/src/utils/phoenix-utils.js
usr/bin/hubs-master/hubs/src/scripts/login.js
at Object. (usr/bin/hubs-master/hubs/src/utils/ phoenix-utils.js:5:1)
at Generator.next () {
code: 'MODULE_NOT_FOUND',
requireStack: [
'usr/bin/hubs-master/hubs/src/utils/ phoenix-utils.js',
usr/bin/hubs-master/hubs/src/scripts/login.js

]
}

npm -v
8.11.0
node -v
16.16.0

sudo yum info nodejs
Loaded plugins: extras_suggestioms, langpacks, priorities, update-motd
Installed Packages
Name : nodejs
Arch : x86_64
Epoch : 2
Version : 12.22.12

Note: "store-instance.ts" is located in 'usr/bin/hubs-master/hubs/src/utils/store-instance.ts'

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
     "lint:js": "eslint '*.js' 'scripts/**/*.js' 'src/**/*.js'",
     "lint:html": "htmlhint 'src/**/*.html' && node scripts/indent-linter.js 'src/**/*.html'",
     "lint": "npm run lint:js && npm run lint:html",
-    "login": "node -r @babel/register -r esm -r ./scripts/shim scripts/login.js",
+    "login": "node -r @babel/register -r ./scripts/babel-register.js -r esm -r ./scripts/shim scripts/login.js",
     "logout": "node ./scripts/logout.mjs",
     "deploy": "node ./scripts/deploy.mjs",
     "undeploy": "node ./scripts/undeploy.mjs",

--- a/scripts/babel-register.js
+++ b/scripts/babel-register.js
@@ -1,0 +1,3 @@
+require("@babel/register")({
+  extensions: [".js", ".jsx", ".es6", ".es", ".mjs", ".ts", ".tsx"]
+});

